fixtures/load.js

```diff
@@ -0,0 +1,177 @@
+import fs from 'fs';
+import path from 'path';
+import crypto from 'crypto';
+import { MongoClient, ObjectId } from 'mongodb';
+import { promisify } from 'util';
+import { PASSWORD } from '../server/constants/authFactorTypes';
+
+const randomBytes = promisify(crypto.randomBytes);
+const pbkdf2 = promisify(crypto.pbkdf2);
+const readFileAsync = promisify(fs.readFile);
+
+const connectDb = async (uri) => {
+  const client = await MongoClient.connect(uri, { useNewUrlParser: true });
+  return client;
+}
+
+const getAdminRole = async (db) => {
+  const adminRole = db.collection('roles').findOne({ name: 'admin' });
+  if (!adminRole) {
+    throw new Error('Admin Role does not already exist in the database, please make sure you have run all the correct migrations');
+  }
+  return adminRole;
+}
+
+const saveAdminUser = async (db, user) => {
+  const salt = await randomBytes(128);
+  const iterations = 100000; // ~0.3 secs on Macbook Pro Late 2011
+  const digestedPassword = await pbkdf2(user.password, salt, iterations, 128, 'sha512');
+  await db.collection('authFactors').insertOne({
+    user: ObjectId(user.id),
+    password: digestedPassword,
+    type: PASSWORD,
+    salt,
+    iterationCount: iterations,
+    isFixture: true,
+  });
+
+  const adminRole = await getAdminRole(db);
+  db.collection('orgMemberships').insertOne({
+    userId: ObjectId(user.id),
+    orgId: null,
+    roles: [{
+      id: adminRole._id,
+    }],
+    isFixture: true,
+  });
+}
+
+const saveUsers = async (db, users) => {
+  let bulk = db.collection('users').initializeUnorderedBulkOp();
+  users.forEach((user) => {
+    // later used for identifying all fixture data
+    user.isFixture = true;
+    user.createdAt = new Date();
```

Comment:
We might also want to mock the createdAt / updatedAt timestamps in the future to display various dates in the front-end. Worth discussing.